tests/unit/lms/views/decorators/lis_result_sourcedid_test.py

```diff
@@ -46,8 +46,9 @@ def test_it_raises_if_validation_raises_other_than_ValidationError(
 
         with pytest.raises(TypeError, match="foo"):
             upsert_lis_result_sourcedid(context, pyramid_request)
-            lis_result_sourcedid_svc.upsert.assert_not_called()
-            wrapped.assert_called_once_with(context, pyramid_request)
```

Comment:
Wat? These lines could never be reached as the first raises an Exception. I've moved them down and guessed at what they were supposed to do.